shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2536,6 +2577,28 @@ void WebContents::SetWebRTCIPHandlingPolicy(
   web_contents()->SyncRendererPrefs();
 }
 
+const std::string WebContents::GetWebRTCUDPPortRangePolicy(void) const {
+  return ToPortRange(
+      web_contents()->GetMutableRendererPrefs()->webrtc_udp_min_port,
+      web_contents()->GetMutableRendererPrefs()->webrtc_udp_max_port);
+}
+
+void WebContents::SetWebRTCUDPPortRangePolicy(
+    const std::string& webrtc_udp_port_range_policy) {
+  uint16_t port_min = 0;
+  uint16_t port_max = 0;
```

Comment:
Given we'll never use these variables if the conditional on line 2591 evaluates to false, they should be moved below the conditional.